.github/workflows/connector_integration_test_single_dagger.yml

```diff
@@ -6,10 +6,14 @@ on:
       test-connectors-options:
         description: "Options to pass to the 'airbyte-ci connectors test' command"
         default: "--modified"
-  # pull_request:
-  #   paths:
-  #     - "airbyte-integrations/connectors/**"
-  #     - ".github/workflows/connector_integration_test_single_dagger.yml"
+  pull_request:
+    paths:
+      #- "airbyte-integrations/connectors/**"
+      #- ".github/workflows/connector_integration_test_single_dagger.yml"
+    types:
+      - opened
+      - synchronize
+      - ready_for_review
```

Comment:
The pipeline will run when:
* the pr is opened
* when a commit is pushed to the pr
* when the pr is made ready for review

But the tests will run only if the PR is not in draft mode thank to [this change](https://github.com/airbytehq/airbyte/pull/26504/files#diff-956ec778a0cbdb8fbf60e4327fc8b80c3c8f06739870ad4f56c5e822203464bfR166)